src/pynwb/tests/integration/test_photometry.py

```diff
@@ -10,7 +10,7 @@
     FibersTable,
     PhotodetectorsTable,
     ExcitationSourcesTable,
-    DeconvolvedRoiResponseSeries,
+    DeconvolvedFiberPhotometryResponseSeries,
```

Comment:
Looks like we need to update these tests -- probably best in a separate PR

Reply:
yes, better in a separate PR